tiledb/sm/compressors/dd_compressor.cc

```diff
@@ -109,14 +108,16 @@ Status DoubleDelta::compress(
       return DoubleDelta::compress<uint8_t>(input_buffer, output_buffer);
     case Datatype::FLOAT32:
     case Datatype::FLOAT64:
+    case Datatype::GEOM_WKB:
+    case Datatype::GEOM_WKT:
       return LOG_STATUS(Status_CompressionError(
-          "Cannot compress tile with DoubleDelta; Float "
-          "datatypes are not supported"));
+          "DoubleDelta tile compression is not yet supported for float or "
+          "geometric datatypes."));
   }
```

Comment:
As I understand it, `GEOM_WKB` and `GEOM_WKT` are just special instances of `BLOB` and we support the double delta filter for `BLOB`, we should support it for the geometric datatypes too. However, I might be missing some context for why this decision was made.